package.json

```diff
@@ -1,54 +1,19 @@
 {
   "name": "baby-project-web",
-  "version": "0.1.0",
-  "private": true,
+  "version": "0.0.0",
```

Comment:
Maybe keept the minor 1 like it had before

Reply:
👍